package.json

```diff
@@ -13,17 +13,35 @@
         "dev": "docz dev",
         "build": "docz build",
         "serve": "docz serve",
-        "lint": "true"
+        "spellcheck": "mdspell './pages/**/*.md' 'CODEOFCONDUCT.md'  --en-US --ignore-numbers --report",
+        "spellfix": "mdspell './pages/**/*.md' './*.md' '!./LICENSE.md'  --en-US --ignore-numbers",
+        "lint-md": "remark ./pages/**/*.md",
+        "lint": "npm run lint-md && npm run spellcheck"
     },
     "dependencies": {
         "@emotion/react": "^11.1.1",
         "@emotion/styled": "^11.0.0",
         "docz": "^2.3.1",
+        "markdown-spellcheck": "^1.3.0",
```

Comment:
Should these be devDependencies?

Reply:
Yes they should. I moved them to deps b/c I was trying to debug the ci failure.  I didn't see this before Rose's approval. I'll file a follow up.